notebooks/229-distilbert-sequence-classification/229-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,319 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4794f066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "onnx_model = \"distilbert.onnx\"\n",
+    "MODEL_DIR = \"model/\"\n",
+    "MODEL_DIR = f\"{MODEL_DIR}\"\n",
+    "onnx_model_path = Path(MODEL_DIR) / onnx_model\n",
+    "dummy_model_input = tokenizer(\"This is a sample\", return_tensors=\"pt\")\n",
+    "torch.onnx.export(\n",
+    "    model,\n",
+    "    tuple(dummy_model_input.values()),\n",
+    "    f=onnx_model,\n",
+    "    input_names=['input_ids', 'attention_mask'],\n",
+    "    output_names=['logits'],\n",
+    "    dynamic_axes={'input_ids': {0: 'batch_size', 1: 'sequence'},\n",
+    "                  'attention_mask': {0: 'batch_size', 1: 'sequence'},\n",
+    "                  'logits': {0: 'batch_size', 1: 'sequence'}},\n",
+    "    do_constant_folding=True,\n",
+    "    opset_version=13,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fff79fc",
+   "metadata": {},
+   "source": [
+    "# Model Optimizer\n",
+    "\n",
+    "[Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0f48c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer_command = f\"mo \\\n",
+    "    --input_model {onnx_model} \\\n",
+    "    --output_dir {MODEL_DIR} \\\n",
+    "    --model_name {checkpoint} \\\n",
+    "    --input input_ids,attention_mask \\\n",
+    "    --input_shape [1,128],[1,128]\"\n",
+    "! $optimizer_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "OpenVINO™ Runtime uses [Infer Request](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Infer_request.html) mechanism which allows running models on different devices in asynchronous or synchronous manners. The model graph is sent as an argument to the OpenVINO API and an inference request is created. The default inference mode is AUTO but it can be changed according to requirement and hardwares available. You can explore the different inference modes and their usage [here.](https://docs.openvino.ai/latest/openvino_docs_Runtime_Inference_Modes_Overview.html)"
```

Comment:
```suggestion
    "OpenVINO™ Runtime uses the [Infer Request](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Infer_request.html) mechanism which allows running models on different devices in asynchronous or synchronous manners. The model graph is sent as an argument to the OpenVINO API and an inference request is created. The default inference mode is AUTO but it can be changed according to requirements and hardware available. You can explore the different inference modes and their usage [in documentation.](https://docs.openvino.ai/latest/openvino_docs_Runtime_Inference_Modes_Overview.html)"
```